flutter/lib/src/screenshot/screenshot_attachment_processor.dart

```diff
@@ -0,0 +1,36 @@
+import 'dart:async';
+
+import 'package:sentry/sentry.dart';
+import '../integrations/native_app_start_integration.dart';
+import '../sentry_widget.dart';
+import 'screenshot_attachment.dart';
+
+class ScreenshotAttachmentProcessor implements SentryClientAttachmentProcessor {
```

Comment:
The name processor here is not ideal because the SDK already has `EventProcessors` and its not an `EventProcessors`, wondering if we can come up with something different.

Reply:
See https://github.com/getsentry/sentry-dart/pull/1088#discussion_r1004358373